tests/testthat/test-tab_afun_cfun.R

```diff
@@ -0,0 +1,256 @@
+                # xxx with 2 should work
+                format = "xx.x")
+        # analyze(vars = c("BMRKR1", "BMRKR1"), 
+        #         afun = list("F" = mean, "SD" = min),
+        #         show_labels = "hidden")
+    
+    lyt %>% build_table(DM_tmp)
+})
+
+test_that(".spl_context contains information about combo counts", {
+    ## Fix for https://github.com/insightsengineering/rtables/issues/517
+    combodf <- tribble(
+        ~valname, ~label, ~levelcombo, ~exargs,
+        "all_X", "All Drug X", c("A: Drug X", "C: Combination"), list(),
+        "all_pt", "All Patients", c("A: Drug X", "B: Placebo", "C: Combination"), list()
+    )
+    
+    n_wrapper_alt_df <- function(alt_counts_df) {
+        function(x, .spl_context, .N_col, .alt_counts_df, .all_col_exprs, 
+                 .all_col_counts, ...) { 
+            
+            cur_col <- paste0(.spl_context$cur_col_split_val[[1]], collapse = ".")
+            
+            # Checks on new .spl_context content
+            stopifnot(.spl_context$cur_col_id[[1]] == cur_col)
+            stopifnot(cur_col %in% names(.spl_context))
+            if (.spl_context$cur_col_split[[1]][1] != "All Patients 2") {
+                stopifnot(all(.spl_context$cur_col_split[[1]] == c("ARM", "COUNTRY")))
+            }
+            
+            if (grepl("all_X", .spl_context$cur_col_id[[1]]) ||
+                .spl_context$cur_col_id[[1]] == "All Patients 2") {
+                in_rows("n" = .N_col, .formats = "xx")
+            } else {
+                AC_colname <- vapply(c("A: Drug X", "C: Combination"),
+                                     function(nmi) {
+                                         paste0(c(nmi, 
+                                                  .spl_context$cur_col_split_val[[1]][2]),
+                                                collapse = ".")
+                                     }, FUN.VALUE = character(1))
+                
+                # Use of cexpr
+                alt_df1c <- .alt_counts_df %>% 
+                    filter(eval(.all_col_exprs[[AC_colname[1]]]))
+                alt_df2c <- .alt_counts_df %>% 
+                    filter(eval(.all_col_exprs[[AC_colname[2]]]))
+                
+                # Normal execution - no use of cexpr
+                alt_df1 <- .alt_counts_df %>% 
+                    filter(ARM == "A: Drug X",
+                           COUNTRY == .spl_context$cur_col_split_val[[1]][2])
+                alt_df2 <- .alt_counts_df %>% 
+                    filter(ARM == "C: Combination",
+                           COUNTRY == .spl_context$cur_col_split_val[[1]][2])
+                
+                # Super manual extraction
+                alt_df1b <- alt_counts_df %>% 
+                    filter(ARM == "A: Drug X",
+                           COUNTRY == .spl_context$cur_col_split_val[[1]][2],
+                           SEX == .spl_context$value[3])
+                alt_df2b <- .alt_counts_df %>% 
```

Comment:
```suggestion
                alt_df2b <- alt_counts_df %>% 
```

Reply:
This looks correct, and suggests a larger change for increased maintainability (good eye/nice debugging to @anajens at catching this as it is).

Please change the argument name to the `n_wrapper_alt_df` function constructor to something easily distinguishable from `.alt_counts_df` (the analysis function argument name), e.g. (`full_alt_df` or `altdf` or `a_counts_df`) Basically anything that doesn't differ by only the smallest printed glyph (.).

Reply:
renamed to `.alt_df` as it is not full and does not contain counts